automation-tests/test/specs/anInitialSuite.e2e.ts

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { step } from 'mocha-steps';
+import {
+  ScratchOrg
+} from '../scratchOrg';
+import {
+  utilities
+} from '../utilities';
+
+/*
+anInitialSuite.e2e.ts is a special case.  We want to validate that the Salesforce extensions and
+most SFDX commands are not present at start up.
+
+We also want to verify that after a project has been created, that the Salesforce extensions are loaded,
+and that the SFDX commands are present.
+
+Because of this requirement, this suite needs to run first before the other suites.  Since the
+suites run in alphabetical order, this suite has been named so it runs first.
+*/
+
+describe('An Initial Suite', async () => {
+  let scratchOrg: ScratchOrg;
+
+  step('Verify our extensions are not initially loaded', async () => {
+    const workbench = await (await browser.getWorkbench()).wait();
+    await utilities.executeQuickPick(workbench, 'Developer: Show Running Extensions');
+    await utilities.pause(2);
+
+    const extensionNameDivs = await $$('div.name');
+    let sfdxKeywordWasFound = false;
+    let salesforceKeywordWasFound = false;
+    for (const extensionNameDiv of extensionNameDivs) {
+      const text = await extensionNameDiv.getText();
+
+      if (text.includes('sfdx')) {
+        sfdxKeywordWasFound = true;
+        utilities.log(`AnInitialSuite - extension ${text} was present, but wasn't expected`);
+      } else if (text.includes('salesforce')) {
+        if (text !== 'salesforce.system-tests') {
+          // salesforce.system-tests is expected, anything else is an issue.
+          salesforceKeywordWasFound = true;
+          utilities.log(`AnInitialSuite - extension ${text} was present, but wasn't expected before the extensions loaded`);
+        }
+      }
+    }
+    expect(sfdxKeywordWasFound).toBe(false);
+    expect(salesforceKeywordWasFound).toBe(false);
+  });
+
+  step('Verify the default SFDX commands are present when no project is loaded', async () => {
+    const workbench = await (await browser.getWorkbench()).wait();
+    const prompt = await utilities.openCommandPromptWithCommand(workbench, 'SFDX:');
+
+    let quickPicks = await prompt.getQuickPicks();
+    let unexpectedSfdxCommandWasFound = false;
+    for (const quickPick of quickPicks) {
+      const label = await quickPick.getLabel();
+      switch (label) {
+        // These three commands are expected to always be present,
+        // even before the extensions have been loaded.
+        case 'SFDX: Create and Set Up Project for ISV Debugging':
+        case 'SFDX: Create Project':
+        case 'SFDX: Create Project with Manifest':
```

Comment:
I think the comment was left elsewhere but would there be value in making the command strings in this file into constants?